docs/modules/plugins/pages/plugin-web-app-to-rest-api.adoc

```diff
@@ -92,3 +92,5 @@ Examples:
 Examples:
 {transformer=FROM_HEADLESS_CRAWLING, column=page-url}
 ----
+==== Required envinroment properties
+* `web-application.main-page-url` - defines main application page name
```

Comment:
1. Put this info before the example
1. Add this info to each affected table transformer